docs/websites/erp/install-an-odoo-11-stack-on-ubuntu-16-04/index.md

```diff
@@ -13,6 +13,7 @@ title: 'Install an Odoo 11 Stack on Ubuntu 16.04'
 contributor:
   name: Damaso Sanoja
   link: https://twitter.com/damasosanoja
+aliases: ['websites/cms/install-an-odoo-11-stack-on-ubuntu-16-04/']
```

Comment:
Oh dear, that are the alias to old URL.

Reply:
we set the old URL in the frontmatter under the `aliases` property, and Hugo uses that information to build a redirect from that URL to the new location